### ISSUE TITLE
Json+Syslog [cef-formatted] integration for analysis results

### ISSUE THREAD

**Issue by jpsenior** [cuckoobox/cuckoo#659]
Build out a small system to add in syslog output for high level signature findings including criticality:
e.g. "Detects VirtualBox through the presence of a Device".

This capability will be added to the reporting modules.

**Comment by jpsenior** [comment]
Note: Intention of this ticket is for vendor-agnostic SIEM integration

**Comment by botherder** [comment]
It's an interesting idea, but we should discuss it. That's something that perhaps @jekil could work on.

**Comment by KillerInstinct** [comment]
Yeah this one is kinda complex -- especially for custom rigs. I made a syslog reporting module a while back, that the user had to customize to fit their needs by modifying the module itself. Not ideal, but for parsing out IOCs from signatures, its kind of required.
